qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/VisitNotePage.java

```diff
@@ -22,6 +22,7 @@ public class VisitNotePage extends Page {
 	private static final By UI_MENU_ITEM = By.className("ui-menu-item");
 	private static final By PRIMARY_DIAGNOSIS_ELEMENT = By.xpath("//ul[1]/li/span/div/strong");
 	private static final By SECONDARY_DIAGNOSIS_ELEMENT = By.xpath("//ul[2]/li/span/div/strong");
+	private static final By SAVED_PRIMARY_DIAGNOSIS_ELEMENT = By.xpath("//*[@id=\"visits-list\"]/li/span[2]");
```

Comment:
Are you able to locate the element with cssSelector?

Reply:
Yeah. I replaced it with a CSS selector. It would be much clear than the xpath.